ext_mongodb.php

```diff
@@ -19,6 +19,108 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+/* == MONITORING ============================================================= */
+namespace MongoDB\Driver\Monitoring;
+
+/* ------ Building blocks for value obejcts */
+abstract class CommandEvent
```

Comment:
Base classes (i.e. CommandEvent and CommandResultEvent) would be an implementation detail for HHVM's code reuse. PHPC likely wouldn't implement these at all. Let's use an underscore prefix and not document these.